src/handlers/rate-limiter.js

```diff
@@ -127,9 +122,7 @@ async function getTokenMetadata (env, authToken, ctx) {
     return decode(cachedValue)
   }
 
-  const accounting = Accounting.create({
-    serviceURL: env.ACCOUNTING_SERVICE_URL
-  })
+  const accounting = ctx.ACCOUNTING_SERVICE ?? Accounting.create({ serviceURL: env.ACCOUNTING_SERVICE_URL })
```

Comment:
**suggestion:** Another reason to always create the `ACCOUNTING_SERVICE` higher up the stack: we don't need to duplicate the logic to create it in each place we use it.